File: kultura/frontNg/src/app/core/models/review.ts
import {Moment} from 'moment-timezone';

export class Review {
  id?: number;
  rating = 0;
  comment = '';
  timeAdded?: Moment;
  culturalOfferingId?: number;
  userId?: number;
  userFirstName?: string;
  userLastName?: string;
  userEmail?: string;
  photos?: number[];
}

File: kultura/DataGenerator/src/main/java/Review.java
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Review {
    String comment;
    int rating;
    long culturalOfferingId;
    long userId;

    @Override
    public String toString() {
        return String.format("INSERT INTO review (comment, rating, time_added, " +
                "cultural_offering_id, user_id) VALUES ('%s', %d, UTC_TIMESTAMP(), %d, %d);\n", comment.replace("'", "\\'"), rating,
                culturalOfferingId, userId);
    }
}
